split-packs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+if [ $# -lt 1 ] || [[ $1 =~ "-" ]] ; then
```

Comment:
Not entirely sure why the origin repo can't have dashes in it. Mine did, so the script failed until I removed the condition.